Documentation/quickstart/install.rst

```diff
@@ -165,22 +165,22 @@ Apache NuttX is actively developed on GitHub. There are two main repositories, `
        $ cd nuttx
        $ curl -L https://github.com/apache/incubator-nuttx/tarball/master -o nuttx.tar.gz 
        $ curl -L https://github.com/apache/incubator-nuttx-apps/tarball/master -o apps.tar.gz
-       $ tar zxf nuttx.tar.gz
-       $ tar zxf apps.tar.gz
+       $ tar zxf nuttx.tar.gz --one-top-level=nuttx --strip-components 1
+       $ tar zxf apps.tar.gz --one-top-level=apps --strip-components 1
        
     There are also ``.zip`` archives available (useful for Windows users): just replace ``tarball`` with
     ``zipball``.
 
   .. tab:: Download stable release
   
     Go to `releases <https://nuttx.apache.org/download/>`_ and choose a version to download. The following
-    example uses version 9.1.0:
+    example uses version 10.1.0:
 
     .. code-block:: console
     
        $ mkdir nuttx
        $ cd nuttx
-       $ curl -L https://downloads.apache.org/incubator/nuttx/9.1.0/apache-nuttx-9.1.0-incubating.tar.gz -o nuttx.tar.gz 
-       $ curl -L https://downloads.apache.org/incubator/nuttx/9.1.0/apache-nuttx-apps-9.1.0-incubating.tar.gz -o apps.tar.gz
-       $ tar zxf nuttx.tar.gz
-       $ tar zxf apps.tar.gz
+       $ curl -L https://downloads.apache.org/incubator/nuttx/10.1.0/apache-nuttx-10.1.0-incubating.tar.gz -o nuttx.tar.gz 
+       $ curl -L https://downloads.apache.org/incubator/nuttx/10.1.0/apache-nuttx-apps-10.1.0-incubating.tar.gz -o apps.tar.gz
+       $ tar zxf nuttx.tar.gz --one-top-level=nuttx --strip-components 1
```

Comment:
Is this actually needed for the apache releases or just the GitHub generated ones?

Reply:
Also url should be updated to the new style download link

Reply:
> Is this actually needed for the apache releases or just the GitHub generated ones? The release validation script does not use this, so I am fairly certain this change is not needed here.

Done!

Reply:
> Also url should be updated to the new style download link

It is better to wait ASF put everything in place first. The download is not working:
curl -L https://www.apache.org/dyn/closer.lua/incubator/nuttx/10.1.0/apache-nuttx-10.1.0-incubating.tar.gz -o nuttx.tar.gz

Do you think we should use the CDN link: https://dlcdn.apache.org/incubator/nuttx/10.1.0/apache-nuttx-10.1.0-incubating.tar.gz ?